examples/tutorial/lessons/lesson-05-actions.adoc

```diff
@@ -0,0 +1,348 @@
+
+== Lesson 05 - Actions
```

Comment:
In this intro of Actions I miss some part just describing that actions are de-coupled from triggers.
Meaning that you define an action and that is re-used by several triggers.
It is implictly infered by the examples but I think a note should help to understand it.